hmac_authentication/hmacauth.py

```diff
@@ -1,57 +1,65 @@
 # -*- coding: utf-8 -*-
 
 from hmac_authentication import exceptions
+
 import base64
+import collections
+import enum
```

Comment:
Er, backported isn't the name of the package, it's just an adjective. The package is called enum: https://pypi.python.org/pypi/enum.